src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,101 @@
+.product-card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  padding: 16px;
+  display: flex;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
+    margin-right: 3px;
+    margin-top: 16px;
+  }
+
+  &__title {
+    display: flex;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 40px;
+    margin-bottom: 0;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    width: 98px;
+    height: 14px;
+    font-weight: 400;
+    font-size: 10px;
+    margin-right: 68px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__rating {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 16px;
+    margin: 0;
+    color: #060b35;
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    margin-top: 24px;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__lable {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
```

Comment:
There is a typo in the class name `&__lable`. It should be `&__label` to match the correct spelling of 'label'. This will ensure consistency and prevent potential issues with styling not being applied correctly.